README-zh.md

```diff
@@ -877,6 +877,7 @@ Awesome Mac
 * [IntelliDock](https://mightymac.app/intellidock/) - 自动隐藏 Dock。
 * [Moom](http://manytricks.com/moom/) - 多任务多窗口的软件。
 * [Magnet](http://magnet.crowdcafe.com/) - 一个窗口管理器，可以保持工作空间的组织。 [![App Store][app-store Icon]](https://itunes.apple.com/cn/app/id441258766)
+* [rcmd](https://lowtechguys.com/rcmd/) - 使用 <kbd>⌘ Right Command</kbd> 键根据名称切换应用程序。 [![App Store][app-store Icon]](https://apps.apple.com/us/app/rcmd-app-switcher/id1596283165)
```

Comment:
```diff
- https://apps.apple.com/us/app/rcmd-app-switcher/id1596283165
+ https://apps.apple.com/cn/app/rcmd-app-switcher/id1596283165
```

Reply:
@alin23

Reply:
@jaywcjlove fixed, thanks for catching it!